GameDefinition/Content/CaveKind.hs

```diff
@@ -118,7 +120,7 @@ arena = rogue
                     , ("tiny lift", 1) ]
   , cstairAllowed = [ ("walled staircase", 20), ("closed staircase", 80)
                     , ("tiny staircase", 1) ]
-  , cdesc         = ""
+  , cdesc         = "Debris litters the wide streets and all the stalls have their shutters down."
```

Comment:
Now that I looked closer, it makes perfect sense: these are not the `rogue` decks with 1 meter wide corridors as I mistakenly assumed, but wide `arena` areas with much wider passages, even though the brightly lit pathways that pedestrians (now absent) walk along are only 1-meter wide. And you got the "stalls" right (and they are indeed shuttered) --- recreation includes shopping, see all the commercial places marked `arena` in 

https://github.com/AllureOfTheStars/Allure/blob/5c0ff182def02055139cef8e40aab1475e9cda77/GameDefinition/Content/PlaceKind.hs

Well done.